app/components/champ-card.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Center, Flex, Heading } from "@chakra-ui/react";
+import ChampIcon from "./champ-icon";
+import champions from "../_data/champions";
+
+function ChampCard({ championId }: { championId: string }) {
+  return (
+    <Flex>
+      {
+        championId
+          ? (
+            <>
+              < Center pe={2} >
```

Comment:
Some cleanup suggestions
```suggestion
              <Center pe={2}>
```